src/main/java/com/internet/shop/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.internet.shop.dao.impl;
+
+import com.internet.shop.dao.OrderDao;
+import com.internet.shop.db.Storage;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.ShoppingCart;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order(shoppingCart.getUserId());
+        Storage.addOrder(order);
+        return order;
```

Comment:
also you need to clear shopCart up to Checklist

Reply:
Sure